src/pages/UserEnrollments/index.js

```diff
@@ -0,0 +1,102 @@
+import React, { useState, useEffect } from 'react';
+import PropTypes from 'prop-types';
+import { groupBy } from 'lodash';
+
+import AddIcon from '../../components/icons/add';
+
+import Spinner from '../../components/icons/spinner';
+
+import Enrollment from './Enrollment';
+
+import './UserEnrollments.css';
+
+import { getUserEnrollmentsBis } from '../../lib/services';
+
+const UserEnrollments = ({ history }) => {
+  const [isLoading, setIsLoading] = useState(true);
+  const [enrollmentsByOrganization, setEnrollmentsByOrganization] = useState();
+
+  useEffect(() => {
+    const onFetchData = async () => {
+      setIsLoading(true);
+      const enrollments = await getUserEnrollmentsBis();
+
+      const enrollmentsByOrganization = groupBy(
+        enrollments,
+        e => e.nom_raison_sociale
```

Comment:
we should rely on the siret rather than the nom_raison_sociale for treatment like this.